CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### App Center
 
 * **[Fix]** Update `Newtonsoft.Json` dependency to version `13.0.2`. The update fixes [GHSA-5crp-9r3c-p9vr](https://github.com/advisories/GHSA-5crp-9r3c-p9vr).
+* **[Fix]** Use `DispatcherUnhandledException` instead of `AppDomain.CurrentDomain.UnhandledException` for WPF applications for postcrash processing. It`s fix [#1727](https://github.com/microsoft/appcenter-sdk-dotnet/issues/1727).
```

Comment:
```suggestion
* **[Fix]** Use `DispatcherUnhandledException` instead of `AppDomain.CurrentDomain.UnhandledException` for WPF applications for postcrash processing. It fixes #1727.
```